vm-setup/roles/packages_installation/tasks/ubuntu_required_packages.yml

```diff
@@ -39,61 +39,21 @@
         name: "*"
         state: latest
 
-    - name: Podman
-      block:
-        - name: Install podman
-          apt: 
-            name: podman
-            state: present
-
-        - name: Registries configuration for podman
-          blockinfile:
-            path: /etc/containers/registries.conf
-            block: |
-                [registries.insecure]
-                registries = ['{{ REGISTRY }}']
-      become: yes
-      when: CONTAINER_RUNTIME == "podman"
-
-    - name: Install docker
-      block:
-        - name: Add Docker’s GPG key
-          apt_key:
-            url: https://download.docker.com/linux/ubuntu/gpg
-            state: present
-
-        - name: Add Docker Repository
-          apt_repository:
-            repo: "deb https://download.docker.com/linux/ubuntu {{ ansible_distribution_release }} stable"
-            state: present
-
-        - name: Update all packages to their latest version
-          apt:
-            name: "*"
-            state: latest
-
-        - name: Install docker
-          apt: name={{ item }} state=latest update_cache=yes
-          loop: [ 'docker-ce', 'docker-ce-cli', 'containerd.io' ]
-
-        - name: Template daemon.json to /etc/docker/daemon.json
-          template:
-            src: "{{ DAEMON_JSON_PATH }}/daemon.json"
-            dest: /etc/docker/daemon.json
-            owner: root
-            group: root
-
-        - name: Restart docker systemd service
-          service:
-            name: docker
-            state: restarted
-            daemon_reload: yes
+    - name: Install podman
+      apt: 
+        name: podman
+        state: present
 
-        - name: Add current user to the docker group
-          user:
-            name: "{{ lookup('env','USER') }}"
-            groups: docker
-            append: yes
-      when: CONTAINER_RUNTIME == "docker"
-      become: yes
-  become: yes
\ No newline at end of file
+    - name: Registries configuration for podman
+      blockinfile:
+        path: /etc/containers/registries.conf
+        create: true
+        block: |
+            unqualified-search-registries = ['registry.nordix.org']
+            
+            [[registry]]
+            prefix = "\registry.nordix.org\"
```

Comment:
I think we don't need backslashes here since this for source cluster and it will be added by Ansible not cloud-init.

Reply:
removed